test/models/work_test.rb

```diff
@@ -0,0 +1,74 @@
+require "test_helper"
+
+describe Work do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+
+  describe "validations" do
+    before do
+      @album = works(:album)
+    end
+    it "is vaild when all fields are present" do
+      assert(@album.valid?)
```

Comment:
This is really assert-style testing.  It works, but it's not the style we're using in class.

```suggestion
      expect(@album.valid?).must_equal true
```